jcli/src/jcli_app/vote/committee_communication_key.rs

```diff
@@ -0,0 +1,26 @@
+use chain_vote::gang::GroupElement;
+use chain_vote::gargamel::PublicKey;
+use chain_vote::{
+    committee::MemberSecretKey, MemberCommunicationKey, MemberCommunicationPublicKey, MemberState,
+};
+use rand_chacha::rand_core::SeedableRng;
+use rand_chacha::ChaCha20Rng;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct CommitteeCommunicationKey {}
+
+impl CommitteeCommunicationKey {
+    pub fn exec(&self) -> Result<(), super::Error> {
+        let key = generate_committee_member_keys();
+        println!("Private: {}", hex::encode(key.to_bytes()));
+        println!("Public: {}", hex::encode(key.to_public().to_bytes()));
```

Comment:
I went for the shortest path here. But I'm not sure if what is the standard we use for outputting data. Will check it too.